website/docs/feature-flag-tutorials/use-cases/scaling-unleash.mdx

```diff
@@ -0,0 +1,323 @@
+
+<details>
+  <summary>Self-hosted observability</summary>
+
+    - **Health checks**: Poll `/health` endpoints on API and Edge instances. Monitor `/internal-backstage/prometheus` on Edge.
+    - **API/Edge instances**: Track CPU, memory, network, request latency, and error rates.
+    - **Database**: Monitor key performance indicators such as CPU utilization, memory usage, disk I/O, network throughput, active connections, query latency/throughput, and available disk space.
+    - **Application logs**: Enhance default logging and centralize (e.g., CloudWatch Logs, Datadog, Splunk).
+    - **Alerting**: Configure alerts based on critical thresholds: high error rates, sustained high latency, unhealthy instances (failing health checks), low disk space (database or instance), high database replication lag, high resource utilization (CPU/memory).
+</details>
+
+Unleash is architected with scalability, performance, and resilience in mind, designed specifically for supporting demanding enterprise workloads. With fast, local feature flag evaluations, Unleash Edge for added scalability, and flexible deployment models, organizations can achieve reliable, high-performance feature management at a global scale, enabling faster, safer software delivery.
+
+## Frequently asked questions (FAQs)
+This FAQ section addresses common questions about scaling Unleash for enterprise use, focusing on performance, high availability, and managing large user and flag volumes.
+
+#### Should I choose cloud-hosted or self-hosted Unleash?
+Choosing a [hosting option](/understanding-unleash/hosting-options) depends on several factors such as: your organization's tolerance for operational overhead, requirements for data residency and infrastructure control, regulatory compliance needs (such as FedRAMP), available SRE/DevOps resources and expertise, and desired speed of implementation (time-to-value).
+
+We generally recommend Unleash Cloud for most enterprises due to reduced operational complexity, built-in high-availability and disaster recovery, and managed scaling by Unleash experts. Choose self-hosted or hybrid if specific control, network isolation (air-gapped), or strict regulatory requirements mandate it.
+
+#### How does Unleash ensure fast feature flag checks without slowing down user experience?
+Unleash achieves speed primarily through how feature flag evaluation works in the [SDKs](/reference/sdks).
+
+- **Local evaluation**: Your application integrates an Unleash SDK. This SDK periodically fetches flag configurations from the Unleash API or Unleash Edge. In server-side SDKs, the evaluation of which flags apply to a specific user happens instantly within your application's memory using the fetched rules and the context you provide (like user ID, location, etc.). Client-side SDKs can rely on Unleash Edge for fast feature flag evaluation.
```

Comment:
```suggestion
- **Local evaluation**: Your application integrates an Unleash SDK. This SDK periodically fetches flag configurations from the Unleash API or Unleash Edge. In server-side SDKs, the evaluation of which flags apply to a specific user happens instantly within your application's memory using the fetched rules and the context you provide (like user ID, location, etc.). Client-side SDKs can rely on Unleash Edge for fast feature flag evaluation (it evaluates all feature flags with one network round-trip and caches the results).
```
Not sure if this is the best note ever but might be relevant to not slowing down user experience. And maybe it's already covered in the point below :see_no_evil: